reference-data/api/src/main/java/org/bahmni/module/referencedata/labconcepts/mapper/ConceptCommonMapper.java

```diff
@@ -19,6 +19,8 @@
         org.openmrs.Concept openmrsConcept = new org.openmrs.Concept();
         if (conceptMetaData.getExistingConcept() != null) {
             openmrsConcept = conceptMetaData.getExistingConcept();
+        } else if (StringUtils.isNotBlank(conceptCommon.getUuid())){
```

Comment:
@angshu or @djazayeri I had trouble replicating the issue locally (explained it in jira), and my understanding of bahmni's packages/classes is still low, so maybe you can clarify this for me:

the above IT tests test Persisters from package org.bahmni.module.admin.csv.persister
which use Mapper's from package org.bahmni.module.admin.concepts.mapper and not referencedata/labconcepts

see

https://github.com/Bahmni/bahmni-core/blob/master/admin/src/main/java/org/bahmni/module/admin/csv/persister/ConceptPersister.java#L6

so I am wondering if the change here is in the right place. thanks!

Reply:
I'm not familiar with this code either, but from tracing through starting from ConceptPersister.persist this seems like the right change.